pkg/util/meminfo.go

```diff
@@ -161,14 +165,22 @@ func readCgroupMemStat(memStatPath string) (int64, error) {
 
 // GetPodMemStatUsageBytes returns the pod's memory usage quantity (Byte)
 func GetPodMemStatUsageBytes(podCgroupDir string) (int64, error) {
-	podMemStatPath := GetPodCgroupMemStatPath(podCgroupDir)
-	return readCgroupMemStat(podMemStatPath)
+	podPath := GetPodCgroupDirWithKube(podCgroupDir)
+	memStat, err := resourceexecutor.NewCgroupReader().ReadMemoryStat(podPath)
```

Comment:
as discussed, we'd keep the CgroupReader as a member variable of the other module. And these utils like `GetPodMemStatUsageBytes` are marked as DEPRECATED since in most cases we can directly use the CgroupReader.ReadXXX.